var/spack/repos/builtin/packages/neurodamus-neocortex/package.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2013-2018 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+from spack import *
+from spack.pkg.builtin.neurodamus_model import NeurodamusModel
+import shutil
+
+
+class NeurodamusNeocortex(NeurodamusModel):
+    """Neurodamus with built-in neocortex model
+    """
+
+    homepage = "ssh://bbpcode.epfl.ch/sim/models/neocortex"
+    git      = "ssh://bbpcode.epfl.ch/sim/models/neocortex"
+
+    version('develop', git=git, branch='master', submodules=True)
+    version('1.0', tag='1.0', submodules=True)
+
+    variant('v5', default=True, description='Enable support for previous v5 circuits')
```

Comment:
Do we really need to separate v5 mod files? If so, we might need to make them visible in main mod directory (if Werner and other people ask)

Reply:
That's why we have a README explaining what we can do to have enerything together (single ln-s )
That's kind of important because (besides cleaning for future, bla, bla, bla) Thalamus requires only V6, and thus this was a simple way to organize the mods.

Reply:
I hope having the README there is enough, but this separation is valuable I think.

Reply:
Ok, fine for me...lets see what users prefer